components/breadcrumb/index.en-US.md

```diff
@@ -0,0 +1,32 @@
+---
+category: Components
+type: Navigation
+title: Breadcrumb
+---
+
+Breadcrumb displays the current page in the system hierarchy position, and can back to previous.
+
+## When to use
+
+- When the system has more than two hierarchy,
+- When you need to inform the user, "where are you",
+- When you need to back to previous.
```

Comment:
Using the word hierarchy here sounds wrong to me, but I'm not sure of a good alternative at the moment. I'm not quite awake yet.

Ideas, @afc163 ?